core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.x86_64.txt

```diff
@@ -162,8 +162,8 @@ hashicorp/vault-k8s:0.10.0
 vault:1.7.0
 # applications
 epiphanyplatform/ignite:2.9.1
-bitnami/pgpool:4.1.1-debian-10-r29
-brainsam/pgbouncer:1.12
+bitnami/pgpool:4.2.2
```

Comment:
this change is not part of this task, correct? You just put it here in advance before further work. Am i right?

Reply:
correct, can be removed if necessary

Reply:
discussed with @plirglo that it can be added here for now